das-dashboard/backend/shared/utils/remote_scp.py

```
@@ -31,14 +31,7 @@ def ensure_profile(self) -> tuple[str, str]:
     def _prepare_ssh_client(self) -> SSHClient:
         ssh = SSHClient()
         ssh.load_system_host_keys()
-
-        user_known_hosts = os.path.expanduser("~/.ssh/known_hosts")
-        if os.path.exists(user_known_hosts):
-            ssh.load_host_keys(user_known_hosts)
-
-        # Require a known host key; add targets to known_hosts before exporting.
-        ssh.set_missing_host_key_policy(RejectPolicy())
-
+        ssh.set_missing_host_key_policy(AutoAddPolicy())
```

Ruff (0.15.20)

[error] Paramiko call with policy set to automatically trust the unknown host key

(S507)